base/workflow/R/do_conversions.R

```diff
@@ -45,7 +46,7 @@ do_conversions <- function(settings, overwrite.met = FALSE, overwrite.fia = FALS
     
     # IC conversion : for now for ED only, hence the css/pss/site check
     # <useic>TRUE</useic>
-    if (ic.flag) {
+    if (ed.ic.flag) {
```

Comment:
the ic_process has been built and tested with ED but it's not supposed to be for ED only. I feel like your BADM_IC_process  file should be called within the ic_process

Reply:
Sure I can merge them into one.